src/test/resources/features/card.feature

```diff
@@ -13,3 +13,24 @@ Feature: Card actions
       And I select TestCard card
       And I add a checklist with TestCheckList title
      Then the checklist section is displayed on the card details
+
+  Scenario: Assign a date to a card
+    Given I log in as admin user
+     When I select TestBoard board
+      And I select TestCard card
+      And I assign a due date
+        | Date     | Today             |
+        | Time     | One hour from now |
+        | Reminder | None              |
+     Then The due date section is displayed on the card details
+
+  Scenario: Assign card to a member
+    Given I log in as admin user
+     When I select TestBoard board
+      And I select TestCard card
+      And I add a member
+        | George Smith  |
+        | James Cooper  |
+        | Joseph Taylor |
+     Then the member's initials are shown in the card
+#      And the member is checked on the board members list
```

Comment:
Please, cleat or decomment this line.
Add a white line bellow this line, please.

Reply:
Solved